src/File/Akka.Streams.File.Tests/LogRotatorSinkSpec.cs

```diff
@@ -314,7 +314,9 @@ private static T AwaitResult<T>(Task<T> task, TimeSpan? timeout = null)
         {
             try
             {
-                task.Wait(timeout ?? TimeSpan.FromSeconds(3));
+                timeout ??= TimeSpan.FromSeconds(3);
+                if (!task.Wait(timeout.Value))
```

Comment:
Check that the task actually finished within the timeout period, else a deadlock can block the spec.